### ISSUE TITLE
Running a Zero-Shot-Inference with given checkpoint and custom dataset, doesn't run

### ISSUE THREAD

**Issue by Leonardin96** [Pointcept/OpenIns3D#29]
I have tried running the Zero-Shot-Inference with the default Replica vocabulary, the provided checkpoint and a custom dataset, but the process is getting killed after the following warning:
`warnings.warn(
The checkpoint state_dict contains keys that are not used by the model:
  model_ema.{decay, num_updates}
Killed`

I have followed the instructions of the installation file for all three modules. 

**Note:**
This is not happening, when using the yoloworld detector.

**Comment by ZheningHuang** [comment]
Hi, and thank you for your interest in our work.

It sounds like the issue may be related to the mask proposal model. The mask proposal component is primarily adapted from Mask3D, which can occasionally be tricky to use. Here’s my suggestion:

1. First, test Mask3D independently by following the installation instructions provided in its repository. We’ve previously tested it, so it should work as expected.

2. Consider generating the mask proposal first, and then use our Snap/rawRGB + Lookup approach for the search process. Our Mask Proposal Model (MPM) is adapted from [Mask3D](https://github.com/cvg/Mask3D), which we have tested, but if you encounter difficulties, you can generate the mask first with their installation.

Let me know if the issue still persists.
